L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -102,6 +102,18 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         }
     }
 
+    func tapKeyboardKey(_ key: Int) {
+        //let key = self.keyboards.buttons[key]
+        let key = app.keyboards.keys.element(boundBy: key)
+        if app.buttons["Continue"].exists == true {
+            // Attempt to find and tap the Continue button
+            // of the keyboard onboarding screen.
+            app.buttons.staticTexts["Continue"].tap()
+            app.tables["Add Credential"].cells.element(boundBy: 1).tap()
+        }
+        key.tap()
+    }
```

Comment:
nit: Will this function be re-used elsewhere? Does it make sense to place it in an equivalent helper class/object? also, can function name is a bit vague

Reply:
No, not really, this is only used by the screensthots tests and this is just a workaround for some locales